arctic/chunkstore/chunkstore.py

```diff
@@ -48,8 +46,9 @@ def _ensure_index(self):
                                       unique=True, background=True)
 
     @mongo_retry
-    def __init__(self, arctic_lib, chunker=DateChunker()):
+    def __init__(self, arctic_lib, chunker=DateChunker(), serializer=NumpyString()):
```

Comment:
I'm happy to change the names to whatever we want. one is numpy records based, the other is numpy arrays (in string format) based. I could change it to something more reflective of the fact its, at its heart, a numpy array